test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (common.isWindows) {
+  // Create readOnlyMod.js and set to read only
+  const readOnlyMod = path.join(tmpdir.path, 'readOnlyMod');
+  const readOnlyModRelative = path.relative(__dirname, readOnlyMod);
+  const readOnlyModFullPath = readOnlyMod + '.js';
+
+  fs.writeFileSync(readOnlyModFullPath, 'module.exports = 42;');
+  // Removed any inherited ACEs, and any explicitly granted ACEs for the
+  // current user
+  cp.execSync('icacls.exe "' + readOnlyModFullPath +
```

Comment:
May be:
```js
cp.execSync(
  `icacls.exe "${readOnlyModFullPath}" /inheritance:r /remove "%USERNAME%"`);
```

Reply:
I did originally have template literals, but a) I wasn't sure they were supported all the way back to all LTS releases (I see now they are), and b) One long string was exceeded the lint line limit (but I could wrap it as shown above). I can add them back if desired.

Reply:
Personally, I find template literals in these cases more clear, they have less syntactic noise (operators, quotes), but this can be my personal taste, so you can decide freely) We did not set strict linting rule for this, but we had many PRs that have replaced concatenations with templates, so templates may be prevalent style in tests now.